src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -14,6 +14,8 @@ public class HttpWebRequest : WebRequest
     {
         private const int DefaultContinueTimeout = 350; // Current default value from .NET Desktop.
 
+        private readonly byte[] _emptyBytes = new byte[0];
```

Comment:
Can you use ```Array.Empty<byte>()``` instead?  If you can't for some reason, any reason this can't be static?